code/src/components/TaskList.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import styled from "styled-components";
+
+import tasks from "reducers/tasks";
+import { devices } from "styles";
+
+export const TaskWrapper = styled.section`
+  @media ${devices.tablet} {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+`;
+
+export const TaskCard = styled.article`
+  position: relative;
+  display: flex;
+  align-items: center;
+  border: 1px solid #dcdcdc;
+  border-radius: 5px;
+  padding: 15px 10px 10px 15px;
+  margin: 10px;
+  background-color: rgba(255, 255, 255, 0.5);
+
+  @media ${devices.tablet} {
+    width: 70vw;
+  }
+`;
+const DeleteButton = styled.button`
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  background: none;
+  border: none;
+  margin-bottom: 10px;
+  color: black;
+  cursor: pointer;
+
+  &:hover {background-color: rgba(255, 255, 255, 0.7)}
+  &:active {
+    background-color: rgba(255, 255, 255, 0.7)
+  box-shadow: 0 5px #666;
+  transform: translateY(4px);
+}
+
+`;
+
+const TaskText = styled.h2`
+  font-weight: 400;
+  font-size: medium;
+  margin: 10px 5px;
+
+  @media ${devices.tablet} {
+    font-size: large;
+  }
+`;
```

Comment:
Really nice! i was struggling a bit with this that was why i decided to use both css and styled components. Will try to do thees changes. :)

Reply:
Yes this is nice! I just mentioned this just in case you wanted to use this to keep your code consistent. There is nothing wrong with combining both css and styled components :)